<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrometerConfig</class>
 <widget class="QDialog" name="SpectrometerConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>210</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_apply">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>210</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="integrationTime">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>55</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.050000000000000</double>
   </property>
   <property name="maximum">
    <double>60000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.050000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="startPixel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>5</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2047</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start pixel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Integration time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>55</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="integrationDelay">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Integration delay</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of averages</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="stopPixel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2047</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop pixel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numberOfAverages">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>105</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dynamicDarkCorrection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dynamic dark correction</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="forgetPercentage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>155</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Forget percentage</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_highResADC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>High-resolution ADC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>155</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lambdaStart">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>5</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>??? nm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lambdaStop">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>??? nm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
